client/extensions/wp-super-cache/state/stats/schema.js

```diff
@@ -4,7 +4,29 @@ export const statsSchema = {
 	patternProperties: {
 		// Site Id
 		'^\\d+$': {
-			type: 'object'
+			type: 'object',
+			items: {
+				supercache: {
+					type: 'object',
+					items: {
+						cached: { type: 'integer' },
+						cached_list: { type: 'object' },
+						expired: { type: 'integer' },
+						expired_list: { type: 'object' },
+						fsize: { type: 'integer' },
+					}
+				},
+				wpcache: {
+					type: 'object',
+					items: {
+						cached: { type: 'integer' },
+						cached_list: { type: 'object' },
+						expired: { type: 'integer' },
+						expired_list: { type: 'object' },
+						fsize: { type: 'integer' },
+					}
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/14669/commits/4ab72acf529a947392617af52babdedd04653603.